Mixins are not available when using TypeScript

I can't use `$t` and `$tChoice` mixins, TypeScript doesn't compile:
```
error TS2339: Property '$t' does not exist on type '{ $: ComponentInternalInstance; $data: {}; $props: Partial<{}> & Omit<Readonly<ExtractPropTypes<__VLS_TypePropsToRuntimeProps<{ team?: OperatorTeam | undefined; operators: Operator[]; ticketCategories: TicketCategory[]; errors: Record<...>; }>>> & VNodeProps & AllowedComponentProps & ComponentCustomProps, never>; .....'.

66                                 {{ $t("main.save") }}
                                      ~~
```

--- a/package.json
+++ b/package.json
@@ -16,6 +16,7 @@
   "description": "allows to connect your `Laravel` Framework localization files with `Vue`.",
   "main": "dist/cjs/index.js",
   "module": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
     "watch": "tsc --watch",
     "test": "jest",

--- a/src/index.ts
+++ b/src/index.ts
@@ -99,6 +99,16 @@ export const reset = (): void => {
 export const trans_choice = transChoice
 
 /**
+ * Type declarations for `$t` and `$tChoice` mixins.
+ */
+declare module '@vue/runtime-core' {
+  interface ComponentCustomProperties {
+    $t: typeof trans
+    $tChoice: typeof transChoice
+  }
+}
+
+/**
  * The Vue Plugin. to be used on your Vue app like this: `app.use(i18nVue)`
  */
 export const i18nVue: Plugin = {

